src/components/InputTooltip.js

```diff
@@ -109,18 +109,43 @@ class InputTooltip extends Component {
         }
     }
 
+    splitText(textToSplit, query) {
```

Comment:
It's really cool function to unit test ;)

Reply:
I'll setup test env in separate PR.